### ISSUE TITLE
A little bug on lower resolutions

### ISSUE THREAD

**Issue by thamathar** [leggett/simplify#140]
Hi, the browser is at the resolution of Width 1280 the Search bar goes over the order selection (New/Older) and makes it impossible to to a search (since can't click on the search)

[Image of the Bug](https://imgur.com/HKO9Xyj)

**Comment by leggett** [comment]
This looks like the bug that was fixed in v1.2.10. Can you confirm you have that version of Simplify?

Instructions on how to see the version number and force an update if you don't have v1.2.10:
https://kixie.zendesk.com/hc/en-us/articles/235601588-How-to-Force-an-Update-of-Your-Chrome-Extensions

**Comment by leggett** [comment]
Please re-open the issue if you are running the latest version. You might also have to just reload Gmail if you already have the latest version.

**Comment by thamathar** [comment]
Sorry for the late delay, but not is not overlaying each other, but when we press the Search button doesn't do nothing, it will not show the "TextBox" and on the console (CTRL+SHIFT+I) gives this error:

```
Uncaught TypeError: Cannot read property 'getElementsByTagName' of undefined
    at initSearch (script.js:225)
    at initEarly (script.js:771)
```

Don't know if this error as anything to do with this problem.

**Comment by leggett** [comment]
Yup, they JUST changed some classnames again. I happened to be up and was able to fix it (in v1.3.2). I also made the code a little more robust for english (using the aria-labels) but this will continue to break in other languages until I figure out a better way to select these elements. :/

**Comment by thamathar** [comment]
Humm the windows at work is on Portuguese (wasn't me installing the windows and didn't have the time to put this on English) but all the programs that I have installed are on English (VS, Chrome and such) maybe is because the windows is on English?

At home since the OS is in English I didn't notice this bug there
